lib/elasticsearch.ts

```diff
@@ -41,3 +51,29 @@ export function get_indices() {
     return filtered_indices.sort();
   });
 }
+
+// We want to keep 14 days of indices. Let's do 15 so I don't have to think hard about what happens
+// when the time is in UTC or when we're before/after midnight.
+const NUM_DAYS = 15;
+
+function filter_old_indices(current_indices) {
+  return new Promise((resolve, reject) => {
+    const acceptable_indices = [];
+    let today = moment();
+    for (let i = 0; i < NUM_DAYS; i++) {
+      acceptable_indices.push(`logs-${today.format("YYYY.MM.DD")}`);
```

Comment:
Can we assume that indexes will always be grouped by day and always have the same prefix string? There's no world where someone might have like `logstash-2016.04.03` and `logs-2016.04.03`, or something?